guides/v2.1/extension-dev-guide/object-manager.md

```diff
@@ -44,13 +43,10 @@ Direct use of the `create` function prevents type validation and type hinting th
 Object creation is also a separate responsibility that should be moved to a dedicated class such as a [factory]({{ page.baseurl }}/extension-dev-guide/factories.html) or [proxy]({{ page.baseurl }}/extension-dev-guide/proxies.html).
 In most cases, the framework generates these classes automatically during code compilation.
 
-<div class="bs-callout bs-callout-warning" markdown="1">
+{:.bs-callout .bs-callout-warning}
 You may notice in the Magento 2 codebase that some core classes still call the `ObjectManager` directly.
-
 This code needs porting or exist for backward compatibility purposes.
```

Comment:
These two sentences were part of the original `<div>` warning. I don't think they need to be separate paragraphs though.

Reply:
That's why 49 and 51 were deleted, so that they collapse into one paragraph in the warning.